etl_manager/meta.py

```diff
@@ -867,11 +918,8 @@ def read_table_json(filepath, database=None):
     if "glue_specific" not in meta:
         meta["glue_specific"] = {}
 
-    if "personal_data" not in meta:
-        meta["personal_data"] = None
-
-    if "special_category_data" not in meta:
-        meta["special_category_data"] = None
+    if "sensitivity" not in meta:
+        meta["sensitivity"] = None
```

Comment:
`[]` instead of `None`. For consistency with line 435?

Reply:
Good spot 👍